### DISCUSSION TITLE
Capillary pressure units

### DISCUSSION THREAD

**Discussion by L278** [PMEAL/OpenPNM#1934] [Q&A] [answered]
If the diameter of the pores are put in as meters, what are the units of the capillary pressure?

**Comment by jgostick** [comment, marked as answer]
Everything should ideally be in SI units.  So R in m, surface tension in N/m, gives capillary pressure in Pa.  Sticking to SI throughout the code is recommended.  Unfortunately there is not a good way to enforce units with numpy arrays.  All the units packages have some shortcomings that we can't accept.
